lib/opal/builder.rb

```diff
@@ -126,18 +125,16 @@ def append_paths(*paths)
     attr_accessor :processors, :path_reader, :compiler_options,
                   :stubs, :prerequired, :preload
 
-
-
     private
 
     def tree_requires(asset, path)
-      if path.nil? or path.empty?
-        dirname = Dir.pwd
-      else
-        dirname = File.dirname(File.expand_path(path))
-      end
+      dirname = if path.nil? || path.empty?
+                  Dir.pwd
+                else
+                  File.dirname(File.expand_path(path))
+                end
```

Comment:
I'm fine with this change but it might be better to let the choice to the developer to use this condensed form or not.
In this case we could go one step further and use an inline ternary operator.

Reply:
ok, I'll try to change indentation rules to use
``` ruby
lvar = if cond
  stmt1
else
  stmt2
end
```

Reply:
How about
``` ruby
lvar =
  if cond
    stmt1
  else
    stmt2
  end
```

From what I see we can support existing syntax only by disabling some core cops.